codecs/hevc.c

```diff
@@ -1363,6 +1363,26 @@ int hevc_parse_sei
                 sei->mastering_display.max_display_mastering_luminance = lsmash_bits_get( bits, 32 );
                 sei->mastering_display.min_display_mastering_luminance = lsmash_bits_get( bits, 32 );
             }
+            else if( payloadType == HEVC_SEI_ITU_T_T35 )
+            {
+                /* Compare metadata header bytes to expected HDR10+ dynamic metadata header. */
+                if ( payloadSize > 10 )
```

Comment:
We're only checking 7 bytes here?

Reply:
The AV1 OBU payload includes header bytes, so the comparison includes the metadata type byte (0x04 = T.35).

Here the payload type is already known in advance and the first byte is the country code.

There is a comment about that in av1.c.

Reply:
OK. I mostly just wanted the payload length sizes to be consistent. I saw 10 here and also for AV1, but that didn't make sense because AV1 was including the header bytes and this wasn't.

Reply:
That was an error, I had written the AV1 logic first and just copied it directly. I fixed it while going through the other comments.